### ISSUE TITLE
Support for Gradle 8

### ISSUE THREAD

**Issue by sheinin** [jrvansuita/GaussianBlur#7]
The lib didn't work after upgrading to Gradle 8 due to the discontinued support of RenderScript.

Are there plans to update the library?
https://developer.android.com/guide/topics/renderscript/compute
![image](https://user-images.githubusercontent.com/3623465/234544367-a71ca43b-4e36-4788-b7a5-a437579110c9.png)

**Comment by jrvansuita** [comment]
Hello there. Thanks for informing me. I was not aware of it. What are the options left? If RenderScript will not be part of Android anymore I think we will not need it. Probably they will provide a better/handy manner to apply blur over images.

**Comment by jrvansuita** [comment]
This library has now 7 years old. Maybe it's getting near its ending.

**Comment by sheinin** [comment]
This library creates aesthetically better images than the alternative "hoko-blur". It would be superb if it were continued!

**Comment by sheinin** [comment]
This library achieved an aesthetically better result than the currently available Hoko Blur. It is essential for my current project where the quality of blurred views deteriorated from the initial release after the upgrade to Gradle 8.

**Comment by najam-amfnrh9** [comment]
I agree with sheinin, It is providing cool results even in Android 13, but it is lacking when I Request R8,
could you plz update it?

**Comment by tjasmin111** [comment]
They are introducing RenderEffect to replace renderscript. But i couldn't find how to use it to blur images not image view
